### DISCUSSION TITLE
Search index debouncing

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8807] [Ideas]
If the same element/site combo is saved multiple times before the first “Update search indexes” job has a chance to execute, the indexes should only actually be updated for the first job, and subsequent jobs should just bail immediately.

**Comment by ghost** [comment]
Hi, I create a new site, Craft auto copy "Entries" are finished, But these entries not have search indexs, We use code `craft.entries({section: 'caseStudies',type: 'caseStudies',limit: 12,search: {query: platforms:office360}});`, But the length of result is zero,  I try change the value of field in entries, And save, Then the result have content. So I think craft do not build the search indexs for the entries of new site. Is it?

**Comment by brandonkelly** [comment]
@DandeSuen That is unrelated to this enhancement idea. I’ve just created a new issue based on your bug report: #6296. Please follow along from there.

**Comment by sjelfull** [comment]
@brandonkelly Could this be a generic enhancement where you can make it easy to only push a job if one already doesn't exist with the same data passed to it?

**Comment by brandonkelly** [comment]
@sjelfull hm… we may be able to do that for the DB queue driver, but probably not for other drivers.
